DEPS

```diff
@@ -6,7 +6,7 @@ deps = {
   "vendor/requests": "https://github.com/kennethreitz/requests@e4d59bedfd3c7f4f254f4f5d036587bcd8152458",
   "vendor/boto": "https://github.com/boto/boto@f7574aa6cc2c819430c1f05e9a1a1a666ef8169b",
   "vendor/python-patch": "https://github.com/brave/python-patch@d8880110be6554686bc08261766538c2926d4e82",
-  "vendor/omaha":  "https://github.com/brave/omaha.git@adcf41919ce8e86e7b2417ab9f53489a9c7ad9ab",
+  "vendor/omaha":  "https://github.com/brave/omaha.git@704253ab1cb6cade108cd1542d2b670599c6c28b",
```

Comment:
Want to remove the extra whitespace before `"https`?

Reply:
Resolved in b98fc88417. Thanks!